spec/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  subject do
```

Comment:
- [ ] As requested but the previous reviewers, Please make sure that you test the `three_recent_posts` method in your User model since it is required that custom methods in the models should have a corresponding unit test. :pray: